### PR TITLE
Added recipe for pyplis

### PR DESCRIPTION
**Pull request by jgliss**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jgliss** [comment]
@conda-forge/staged-recipes I updated the recipe and now all checks are passing. I would appreciate if you could review the PR. Thanks in advance!

File: recipes/pyplis/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - pyplis.test
+
+about:
+  license: GPL-3.0
+  home: https://github.com/jgliss/pyplis
+  license_family: GPL
+  license_file: LICENSE
+  summary: Python library for the analysis UV SO2 camera data
+  doc_url: https://pyplis.readthedocs.io/
+  dev_url: https://github.com/jgliss/pyplis.git
```
Comment (ocefpaf) [inline recipes/pyplis/meta.yaml]: Can you remove the `.git`.

File: recipes/pyplis/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    - python
+  run:
+    - python
+    - numpy >=1.11
+    - scipy
+    - pandas
+    - astropy
+    - matplotlib >=1.5
+    - basemap
+    - opencv == 3.4.4
```
Comment (scopatz) [inline recipes/pyplis/meta.yaml]: This should be `opencv =3.4.4`

Comment (scopatz) [reply recipes/pyplis/meta.yaml]: And is it really strictly this version number and not `opencv >=3.4.4`

File: recipes/pyplis/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 06d2e58986efab8bb46a4b5ca27e73790686d83967039732b7746f286e009c66
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```
Comment (scopatz) [inline recipes/pyplis/meta.yaml]: Can this be `noarch: python`?

File: recipes/pyplis/meta.yaml (hunk 1 above)
Comment (jgliss) [reply recipes/pyplis/meta.yaml]: @ocefpaf Will fix that

File: recipes/pyplis/meta.yaml (hunk 2 above)
Comment (jgliss) [reply recipes/pyplis/meta.yaml]: @scopatz Thanks, I had problems earlier with newer opencv versions here (circleci checks were failing). I found [this thread](https://github.com/bioconda/bioconda-recipes/issues/13974) where they apparently fixed the opencv version to 3.4.4 (and added jasper as test req.). I will try to leave opencv version open in my next commit, and see if it works as well.

**Comment by jgliss** [comment]
@conda-forge/staged-recipes I applied all suggested changes and all checks are passing. Could you please review the PR once more. Thanks in advance!

File: recipes/pyplis/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    - python
+  run:
+    - python
+    - numpy >=1.11
+    - scipy
+    - pandas
+    - astropy
+    - matplotlib >=1.5
+    - basemap
+    - opencv >= 3.4.4
```
Comment (scopatz) [inline recipes/pyplis/meta.yaml]: This has to be `opencv >=3.4.4` (no second space)

**Comment by chrisburr** [comment]
This wasn't working before, just having the extra whitespace caused the CI to silently fail. See [here](https://circleci.com/gh/conda-forge/staged-recipes/37695?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link).

**Comment by chrisburr** [comment]
Ah it's a OpenGL thing, you can fix it by making a file called `yum_requirements.txt` in the recipe directory as explained [here](http://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=yum_requirements#core-dependency-tree-packages-cdt).

**Comment by jgliss** [comment]
@chrisburr thanks a lot for the hint, so the tests actually never passed. 
@conda-forge/staged-recipes, @conda-forge/opencv I am a little lost here and would appreciate help: checks are failing due to missing libGl library when importing opencv:
```
File "/home/conda/feedstock_root/build_artifacts/pyplis_1558473678815/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/python3.7/site-packages/pyplis/helpers.py", line 29, in <module>
    from cv2 import pyrUp
ImportError: libGL.so.1: cannot open shared object file: No such file or directory
Tests failed for pyplis-1.4.3-pyhd2ac866_0.tar.bz2 - moving package to /home/conda/feedstock_root/build_artifacts/broken
WARNING:conda_build.build:Tests failed for pyplis-1.4.3-pyhd2ac866_0.tar.bz2 - moving package to /home/conda/feedstock_root/build_artifacts/broken
```
I tried to follow [these suggestions](https://github.com/bioconda/bioconda-recipes/issues/13974) but this did not work. Also tried to follow the [conda-forge suggestions](https://conda-forge.org/docs/maintainer/knowledge_base.html#libgl) but this also did not succeed ([see here](https://circleci.com/gh/conda-forge/staged-recipes/37692).

Any help or suggestions is very much appreciated! 

Cheers

**Comment by jgliss** [comment]
@chrisburr Thanks a lot for your hints! [I tried this before](https://circleci.com/gh/conda-forge/staged-recipes/37692) but without success, but I guess this was due to missing yum_requirements.txt in the recipe directory. Will try again now!

**Comment by jgliss** [comment]
@conda-forge/staged-recipes All checks are passing now, could you please review? Thanks in advance.

File: recipes/pyplis/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 06d2e58986efab8bb46a4b5ca27e73790686d83967039732b7746f286e009c66
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
```
Comment (chrisburr) [inline recipes/pyplis/meta.yaml]: You don't need these just the `yum_requirements.txt` file if you're only using something that depends on OpenGL. In which case, I think you can move this back to being `noarch`?

File: recipes/pyplis/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+    - matplotlib >=1.5
+    - basemap
+    - opencv
+    - geonum
+    - pydoas
+    - six
+
+test:
+  imports:
+    - cv2
```
Comment (chrisburr) [inline recipes/pyplis/meta.yaml]: I guess this was just for testing?
```suggestion
```

**Pull request merged**